### PR TITLE
Units manager

### PR DESCRIPTION
**Pull request by zack37**

### CODE REVIEW

**Comment by 311chaos** [comment]
@zack37 - Can you take a look at the int parse stuffs and make sure that is ok? I also merged this all the way to the Integration Hub and saw an issue that we need a test to support, along with a fix. The scenario is:
- Create a new property with the new "Building Category" and "Building Type" values.
- Go back to edit mode on that same property, and the "Building Category" and "Building Type" values are not saved. The fields are reverted back to "Select"

**Comment by 311chaos** [comment]
+1 from me

**Comment by 311chaos** [comment]
@TylerGarlick - Can you review and then pull this in?

File: ll-property-information-edit.html [hunk 1]
```diff
@@ -262,12 +288,26 @@
           value: function() {
```
Comment (TylerGarlick) [inline ll-property-information-edit.html:288]: Does anyone know if we have this data in an api? Or a shared location, because I know we use it across ui's

**Comment by TylerGarlick** [comment]
:+1:

**Pull request merged**